addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/automation.html

```diff
@@ -47,11 +67,13 @@ <H2>YAML definition</H2>
     <pre>
   - type: script
     parameters:
-      action:                    # String: The executed action - available actions: add, remove, run
+      action:                    # String: The executed action - available actions: add, remove, run, enable, disable
       type:                      # String: The type of the script
       engine:                    # String: The script engine to use - can be used to override the default engine for the file extension
       name:                      # String: The name of the script, defaults to the file name
       file:                      # String: The full file path, must be readable
+    scriptParameters:
+      target:                    #String: The URL to be invoked for "targeted" script type
```

Comment:
- This should be under `parameters`.
- `# String`

Reply:
Actually, it was mentioned in the issue. I would also prefer to put it into parameters as we don't have any other scriptParameters.
Should I move it then?

Reply:
Leave as is for now.

Reply:
Apparently I originally suggested `scriptParameters` ;) https://github.com/zaproxy/zap-extensions/pull/3433#discussion_r783811790
But the env already supports "script parameters" in `parameters` (https://www.zaproxy.org/docs/desktop/addons/automation-framework/environment/) so lets standardise on putting them all in `parameters` ?